### ISSUE TITLE
Create mockup for footer

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-selfdeposit#29]

**Comment by maxdmayhew** [comment]
<img width="960" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMXRpQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2ae41c5735754592ffa297adaaaadc2a455ded4b/Open%20Emory%20Footer%20Draft.png" alt="Open Emory Footer Draft.png" />

**Comment by eporter23** [comment]
Looks great @maxdmayhew ! Some work I need to plan for is how many OpenEmory-contextual links will be in the footer, but hopefully your mockup can accommodate 2 or 4 columns depending on how that plays out?

**Comment by eporter23** [comment]
Sounds good, thanks!

**Comment by eporter23** [comment]
@maxdmayhew I tested the footer - looks great, and the links are all correct. One revision: in other apps footers and in the mockup you produced, the links on either side of the shield are not underlined. Can you adjust those link styles to remove the underlining?

**Comment by maxdmayhew** [comment]
Of course, sorry it's just an old habit! could we add that to the samvera/versioning ticket? If not I can do that now.

https://accessibleweb.com/question-answer/do-links-need-to-be-underlined/#:~:text=Although%20not%20technically%20required%20by,a%20link%20from%20regular%20text.

**Comment by eporter23** [comment]
I agree with you @maxdmayhew ! Just aiming for consistency across our apps, but if that's a pattern we should implement across all the footers we can certainly discuss it.
